src/utils/api.jsx

```diff
@@ -39,4 +39,14 @@ export const getComments = (review_id) => {
     .then((res) => {
         return res.data.comments
     })
+}
+
+export const postComment = (review_id, username, body) => {
+    return api.post(`api/reviews/${review_id}/comments`, {username, body})
+    .then((res) => {
+        return res.data.comment
+    })
+    .catch((error) => {
+        console.log(error)
+    })
```

Comment:
careful about catching your errors in your api file.

Once they're caught here that's the end of the story!

If you want to handle the errors in React you'll need to catch them in the component instead

also this file should be a `.js` file!